packages/mantine/src/components/table/table-actions/TableActions.tsx

```diff
@@ -19,14 +58,16 @@ export const TableActions = <T,>({children, ...others}: TableActionsProps<T>): R
             span="content"
             order={TableComponentsOrder.Actions}
             py="sm"
-            classNames={{col: TableActionsClasses.col}}
+            {...ctx.getStyles('actionsRoot', {})}
             {...others}
         >
-            <Group gap="xs" className={TableActionsClasses.wrapper}>
+            <Group gap="xs" {...ctx.getStyles('actionsGroup', {})}>
                 {multiRowSelectionEnabled
                     ? (children as (data: T[]) => ReactNode)(selectedRows)
                     : (children as (datum: T) => ReactNode)(selectedRows[0])}
             </Group>
         </Grid.Col>
     );
 };
+
+TableActions.extend = identity as unknown as CustomComponentThemeExtend<TableActionsFactory>;
```

Comment:
How come this was casted as unknown first? 
![](https://i.giphy.com/51Uiuy5QBZNkoF3b2Z.webp)

Reply:
```identity as kdang as CustomComponentThemeExtend<TableActionsFactory>```

Reply:
Thanks @kxdang, you made me realize I actually fixed the type here https://github.com/coveo/plasma/pull/3620/files#diff-876ae5dce86f809b1052c579f3ba1ddc260bfa3285aaecb7974516037820caecR5 and I don't need to cast to `unknown` anymore!